### DISCUSSION TITLE
幸福感是什麼

### DISCUSSION THREAD

**Discussion by shuxian12** [1101-GS4719-C/Python-Community-GS4719-C#1312] [General]
**「幸福」**(或稱快樂，英文 Happiness)是一個抽象的概念，每個人心中都有 自己的幸福定義和想法。對個人而言，客觀生活環境與遭遇攸關生活幸福與否， 但自身最能判斷個人生活遭遇的幸福程度，因此「主觀幸福感（Subjective Well-being)」扮演了評估美好生活之關鍵角色。

而另外一方面，因為幸福感具有主觀的意識，因此許多研究或評估大多聚焦在客觀幸福感，也就是可以用各種客觀的數值去評斷，最有名的莫過於不丹國王提出的「國民幸福總值」(Gross National Happiness, GNH)，反思既有以 經濟做為國家競爭力指標的作法，認為經濟指標不能表現國民真正的福祉與幸 福，從而提出「國民幸福總值」做為新興的國家競爭力指標。

而主觀幸福感也有許多學者不同的定義，如荷蘭心理學家Veenhoven，他認為幸福感個人對於生活品質的主觀感受，也是個人對於自己生活的整體的評價。主要由兩個面向所組成：
* 享樂水平（hedonic level）：個人對生活的各種影響在情感上令人愉悅的程度和主觀感受，涉及到“享樂基調”(愉快的-不愉快的的維度)，直觀解釋為自己的情緒的感受，如高興、喜悅、憤怒等。
* 滿足感（contentment）：個人對生活中實現目標、願望或生活得到滿足的程度以及個人價值感等主觀感知想法，為自我生活經驗的總體評價，屬於認知方面。

**Comment by alicia0725** [comment]
之前常常會跟朋友說，我因為什麼事情很幸福，而我口中的幸福基本上來自於對生活上小事情的滿足感，反而在一些別人看來幸褔的生活條件上我卻容易抱怨，這可能是因為主觀的享樂基調不同。因此我發現如果要容易感到幸福，應該要降低自己對別人的要求，不是要放棄生活品質，而是不要盲目追求，並且不要過度為難

**Comment by Elaine2899** [comment]
我認為「主觀的幸福」非常重要。每個人會感受到的快樂和悲傷都不同，很多快樂可能來自成就、遊戲、旅行或運動......，這些快樂固然誠實也合理，但我覺得生活需要被「小幸福」包圍，而這些小幸福可能不需要自己努力就能獲得。
以我自己為例，我非常喜歡觀察生活中的小事，像是看到路上有些可愛生物做了有趣的舉動、今天經過了一團漂亮的花圃或是感受不同個性之間的差異，等等不同面向都有可能讓我感受到快樂。(但是當生活變得忙碌，就會很難在生活中觀察有趣的事情，思緒會被事情占滿而沒有空間享受眼前的樂趣。)
